doc/docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,106 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer').themes.github;
+const darkCodeTheme = require('prism-react-renderer').themes.dracula;
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'solid_lints',
+  tagline: 'An opinionated set of lint rules based on industry standards.',
+  url: 'https://solid-software.github.io/',
+  baseUrl: '/solid_lints/',
```

Comment:
We don't really need this, do we?
```suggestion
  baseUrl: '/',
```

Reply:
![Screenshot 2024-01-24 at 15 00 39](https://github.com/solid-software/solid_lints/assets/145997968/0f8298f6-ff89-4a23-b0cd-f75b166098be)

Reply:
This is odd -- I think I used just the slash when making changes recently